source/_posts/2023-06-27-knx-collaboration.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Home Assistant collaborates with KNX"
+description: "Home Assistant collaborates with KNX"
+date: 2023-06-27 00:00:00
+date_formatted: "June 27, 2023"
+author: Uwe Bernitt
+comments: true
+categories:
+  - Announcements
+og_image: /images/blog/2023-06-27-knx-collaboration/social.png
+---
+
+<img src='/images/blog/2023-06-27-knx-collaboration/social.png' alt="The KNX panel in Home Assistant" class='no-shadow'>
+<br><br>
+
+We’re excited to announce that the [KNX Association](https://www.knx.org/knx-en/for-professionals/index.php) has officially endorsed the [KNX integration in Home Assistant](/integrations/knx/). The KNX Association will collaborate with the KNX team at Home Assistant to help with the maintenance of the KNX integration and further streamline the user experience.
+
+KNX is a home automation standard for commercial and residential building automation that works locally. It is mainly, but not exclusively, based on a central data cable which is generally installed when the house is being built. KNX also has extensions to work over radio frequencies or via IP infrastructure. KNX partners with hundreds of smart home manufacturers and thousands of installers.
+
+The KNX integration in Home Assistant allows users to use the power of Home Assistant to unify their KNX devices with other smart home devices and experience a beautiful user interface, powerful automations, and local voice control. Users can also configure Home Assistant to expose non-KNX devices back to the KNX network.
+
+<img src='/images/blog/2023-06-27-knx-collaboration/panel.png' alt="The KNX panel in Home Assistant" class='no-shadow'>
+<br><br>
+
+Home Assistant introduced the KNX integration in [Home Assistant 0.24](https://www.home-assistant.io/blog/2016/07/16/sqlalchemy-knx-join-simplisafe/) in July 2016. Since its introduction 7 years ago, the community has continued to evolve it. The development is currently led by [marvin-w](https://github.com/marvin-w) and [farmio](https://github.com/farmio), who brought the integration to a platinum quality level.
```

Comment:
```suggestion
Home Assistant introduced the KNX integration in [Home Assistant 0.24](/blog/2016/07/16/sqlalchemy-knx-join-simplisafe/) in July 2016. Since its introduction 7 years ago, the community has continued to evolve it. The development is currently led by [marvin-w](https://github.com/marvin-w) and [farmio](https://github.com/farmio), who brought the integration to a platinum quality level.
```